packages/type-system/ts/README.md

```diff
@@ -0,0 +1,18 @@
+# Type-System TypeScript packages
+
+This folder contains
+- [type-system-node](./type-system-node) (_Autogenerated_) - The Type System package, built to target NodeJS
```

Comment:
I'm not sure if we want to commit these to version control

Reply:
(it's worth noting that these are built packages, not plain source code that needs transpiling)

If we don't commit them then I think we'll need to figure out a publishing mechanism before we're able to use these easily in a block. If we do commit it then I think it should be simple to install them by just referencing their link in git

Reply:
We've decided it makes sense to add them to version control. When we come to adding CI checks we'll need to check for a diff between the generated content and the committed one.